bin/phraseapp_updater

```diff
@@ -41,8 +42,10 @@ class PhraseAppUpdaterCLI < Thor
       puts "Bad PhraseApp API key."
     rescue PhraseAppUpdater::PhraseAppAPI::BadProjectIDError => e
       puts "Bad PhraseApp project ID: #{phraseapp_project_id}"
+    rescue PhraseAppUpdater::LocaleFile::BadFileTypeError => e
```

Comment:
It would be cool if you printed out what the valid file types are here. Saves someone from going digging into source if they're not sure.